client/src/store/epics/questions.js

```diff
@@ -40,13 +56,22 @@ export const createQuestion = action$ => action$
   .switchMap(({headers, payload}) => Observable
     .ajax.post('http://localhost:8080/api/question', payload, headers)
     .map(res => res.response)
-    .map(question => ({
-      type: ActionTypes.CREATE_QUESTION_SUCCESS,
-      payload: question,
-    }))
-    .catch(error => Observable.of({
-      type: ActionTypes.CREATE_QUESTION_ERROR,
-      payload: {error},
-    })),
+    .mergeMap(question => Observable.of(
+      {
+        type: ActionTypes.CREATE_QUESTION_SUCCESS,
+        payload: question,
+      },
+      Actions.addNotificationAction(
+        {text: `Question with text "${question.text}" created`, alertType: 'info'},
+      ),
+    ))
+    .catch(error => Observable.of(
+      {
+        type: ActionTypes.CREATE_QUESTION_ERROR,
+        payload: {error},
+      },
+      Actions.addNotificationAction(
+        {text: `[question create] Error: ${ajaxErrorToMessage(error)}`, alertType: 'danger'},
```

Comment:
I have improved the legibility of these three. But I do not know what you mean with this: "Why is this necessary? Can't we just use devtools request throttling for this?"